test/api/vote.test.js

```diff
@@ -243,7 +243,7 @@ describe('Vote API', function() {
                         if (err) return done(err);
 
                         var receivedVote = res.body;
-                        receivedVote.alternative.should.equal(String(vote.alternative));
+                        receivedVote.alternative._id.should.equal(String(vote.alternative));
```

Comment:
Should probably test the populated fields too? I.e. `vote.alternative.election` etc.